### DISCUSSION TITLE
Migration de buzhug vers Python 3

### DISCUSSION THREAD

**Discussion by georges1209** [brython-dev/brython#2517] [General]
Bonjour à tous,
je sais que mon message n'a rien à voir.
Mais j'ai effectué une migration de la base de données buzhug vers Python 3.
Alors si cela intéresse en premier lieu son auteur Pierre Quentel.
Je suis à l'écoute.
Georges

**Comment by PierreQuentel** [comment]
Bonjour Georges,

Je croyais ne plus jamais entendre parler de buzhug, votre message me fait plaisir ! La version Python 3 est sur Github ?

- Pierre

**Comment by georges1209** [comment]
Bonjour Pierre,
Merci pour votre réponse que j'attendais avec impatience.
C'est le seul moyen que j'ai trouvé pour vous contacter.
Votre base de données est tout à fait excellente, parfaitement pythonic.
Je l'ai utilisé pendant de nombreuses années dans un cadre professionnel pour produire des tableaux de bord
en contrôle de gestion.
Je ne suis pas un programmeur professionnel, je fais plutôt du hard programming, mais je me débrouille plutôt bien.
Alors pour la conversion, je passe le module de test sans problème en Python 3.12.0 sous Windows 10 Pro.
Les points essentiels sont le passage en UTF-8, et la traduction des string en binaire, avec quelques particularités résolues grâce au net.
La moindre des choses est de vous en réserver la primeur, et aussi l'usage, reconnaissance du travail accompli.
Je tiens à votre disposition un fichier buzhug3.7z
Mon adresse personnelle est monprénomarobasemonnompointnet.
Georges TUCHOLSKI
